velox/exec/tests/PartitionedOutputTest.cpp

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "velox/exec/PartitionedOutput.h"
+#include <gtest/gtest.h>
+#include "velox/exec/Task.h"
+#include "velox/exec/tests/utils/AssertQueryBuilder.h"
+#include "velox/exec/tests/utils/OperatorTestBase.h"
+#include "velox/exec/tests/utils/PlanBuilder.h"
+
+using namespace facebook::velox;
+using exec::test::AssertQueryBuilder;
+
+namespace facebook::velox::exec::test {
+
+class PartitionedOutputTest : public OperatorTestBase {
+ protected:
+  static StringView generateString(size_t size, char value) {
+    std::string result;
+    result.reserve(size);
+    for (size_t i = 0; i < size; i++) {
+      result += value;
+    }
+    return StringView{result};
```

Comment:
We can't do this? StringView doesn't hold the buffer if it is not inlined? You might need to return std::string here? Not sure if we could use fuzzer vector here.

Reply:
Wow, nice catch. I must've misread. I looked like it does `memcpy`, but you are right. It only does it for inline values.